packages/shared/lib/core/profile-manager/actions/events-handlers/handleNewOutputEvent.ts

```diff
@@ -35,7 +35,7 @@ export async function handleNewOutputEventInternal(
         !get(allAccountActivities)[accountIndex].find((_activity) => _activity.id === output.outputId)
     const isNftOutput = output.output.type === OUTPUT_TYPE_NFT
 
-    if ((account.depositAddress === address && !output?.remainder) || isNewAliasOutput) {
+    if ((account?.depositAddress === address && !output?.remainder) || isNewAliasOutput) {
```

Comment:
why is this required?

Reply:
An error was thrown when testing with ledger, so I added it for safety.